_drafts/oryx/versioning.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Versioning
+description: Versioning
+template: concept-topic-template
+---
+
+# Versioning
+In this document, we outline the methods we use to deliver an advanced application development platform while maintaining stability. Our goal is to ensure that any upcoming changes are introduced in a predictable manner. We want all Oryx users to be informed about the addition of new features and adequately prepared for the removal of outdated ones.
+
+Stability is a critical aspect of any software framework, and it's especially important for frontend frameworks like Oryx. The stability of a framework refers to its ability to function reliably and consistently over time, despite changes in the environment or updates to its components. 
+
+Ensuring that a framework is stable requires careful planning, testing, and versioning. In this article, we will explore how the Oryx framework achieves stability through its versioning and release processes. We'll look at how Oryx manages its releases, how it uses version numbers to indicate changes, and how it ensures backward compatibility for its users. 
+
+## Semantic versioning
+Oryx follows [semantic versioning](https://semver.org/) principles. 
+
+This indicates that with a version number x.y.z:
+- If there are critical bug fixes to be released, a patch release is made by altering the z number (e.g., 1.5.2 to 1.5.3).
+- If new features or non-critical fixes are being released, a minor release is made by modifying the y number (e.g., 1.5.2 to 15.6.0).
+- If there are significant changes that may break compatibility, a major release is made by changing the x number (e.g., 1.5.2 to 2.0.0).
+
+## Release Process
+The Oryx framework comprises three distinct components, each of which is released separately:
+- Libraries
+- Application
+- Labs
+
+### Libraries
+The primary sources of functionality within the Oryx frameworks are the Oryx libraries, which include core features, utilities and helpers, domain-specific functionality, as well as pre-designed themes and presets that can be readily used. 
+All the [packages](//TODO: add link) released together under the same version. 
+
+### Applications
+Oryx applications serve as boilerplate templates for specific domains or areas, such as b2c storefronts, fulfillment applications, and more.
```

Comment:
This seems wrong, let's consult with @supproduction on what this is exactly.

Reply:
Looks like all right

Reply:
Contacted Alexey, but it looks like he don't really know what you're talking about. Can you specify your point @tobi-or-not-tobi ?